data/weapons/weapon_data.csv

```diff
@@ -192,11 +192,11 @@ Plover Rocket Rack,diableavionics_plover_fighter,0,,1220,600,,200,,33,12,10,4,,,
 ,diableavionics_wanzerArm,,,,1,1,,,,,,,,,,,,,10,,,,,,,,,,,,,,SYSTEM,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 #AVALANCHE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Snub Recson,diableavionics_avalancheHead,,,,600,,200,,,45,,,,,KINETIC,,,0,3.5,2,0.5,2,6,5,6,,600,,,100,,,SYSTEM,,,Diable Avionics,,Anti Shield,,,,,,,,,,
-Snow Ball Shotgun,diableavionics_avalancheGun,,,,500,,450,,,30,,,,,HIGH_EXPLOSIVE,,,0,2,,,6,12,0.5,2.5,,1000,,,100,,,"SYSTEM, DO_NOT_CONSERVE, PD, PD_ALSO",,,Diable Avionics,,Shredder,,,,,,,,,,
+Snub Recson,diableavionics_avalancheHead,,,,600,,250,,,45,,,,,KINETIC,,,0,3.5,2,0.5,2,6,5,6,,600,,,100,,,SYSTEM,,,Diable Avionics,,Anti Shield,,,,,,,,,,
+Snow Ball Shotgun,diableavionics_avalancheGun,,,,500,,360,,,30,,,,,HIGH_EXPLOSIVE,,,0,2,,,2,4,0.5,2.5,,1000,,,100,,,"SYSTEM, DO_NOT_CONSERVE, PD, PD_ALSO",,,Diable Avionics,,Shredder,,,,,,,,,,
```

Comment:
Avalanche shotgun set to 360 damage (from 450). Spread (min and max) reduced by 66%.
Avalanche snub recson set to 250 damage (from 200).